packages/babel-register/src/index.ts

```diff
@@ -13,3 +13,6 @@ const node = require("./nodeWrapper");
 const register = node.default;
 
 Object.assign(exports, node);
+
+// make TypeScript happy, by making this file a "module"
+export default register;
```

Comment:
This is an observable change, because it changes the source type (and thus the compiled output) from `"script"` to `"module"`.
I'll revert this file to `.js`; it will go away in Babel 8 anyway.